gym/wrappers/order_enforcing.py

```diff
@@ -9,10 +9,22 @@ def __init__(self, env):
         self._has_reset = False
 
     def step(self, action):
-        assert self._has_reset, "Cannot call env.step() before calling reset()"
-        observation, reward, done, info = self.env.step(action)
-        return observation, reward, done, info
+        assert self._has_reset, "Cannot call env.step() before calling env.reset()"
+        return self.env.step(action)
 
     def reset(self, **kwargs):
         self._has_reset = True
         return self.env.reset(**kwargs)
+
+    def render(self, **kwargs):
+        if hasattr(self.unwrapped, "disable_render_order_enforcing"):
```

Comment:
Do we need this? That seems to be a completely new pattern, and not a very useful one in my opinion. Can't we just make render order enforcing an automatic feature of `OrderEnforcing`, and if someone wants to disable it, they can disable the wrapper

Reply:
Changed to a wrapper parameter in 6c19af22551ff187053205cd7776dea6b8eaca29